src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    private static final int SALARY_SPLIT_INDEX = 2;
+    private static final int NAME_SPLIT_INDEX = 1;
+    private static final int DATE_SPLIT_INDEX = 0;
+    private SimpleDateFormat sdformat = new SimpleDateFormat("dd.MM.yyyy");
+
+    private int salaryCounter(String data) {
+        String[] dataSplit = data.split(" ");
+        return Integer.valueOf(dataSplit[SALARY_SPLIT_INDEX]) * Integer.valueOf(dataSplit[3]);
```

Comment:
Any magic numbers should be constants. The index '3' here should be extracted to a constant field to improve code readability and maintainability.

Reply:
magic number

Reply:
fix magic numbers and magic literals in all places